test/fail/ok/illegal-await.tc.ok

```diff
@@ -1,37 +1,37 @@
-illegal-await.mo:3.36-3.37: type error [M0087], ill-scoped await: expected async type from current scope $@anon-async-3.22, found async type from other scope $top-level
-  scope $@anon-async-3.22 is illegal-await.mo:3.22-3.40
+illegal-await.mo:3.36-3.37: type error [M0087], ill-scoped await: expected async type from current scope $anon-async-3.22, found async type from other scope $top-level
+  scope $anon-async-3.22 is illegal-await.mo:3.22-3.40
```

Comment:
why these changes? 😕

Reply:
That's a side effect of the unused declarations PRs. I think he doesn't want to warn about unreferenced anon ids, and was suppressing the warning for internal ids (@xxx), so just prefixed the anonymous ones with the internal @-marker. @luc-blaeser, is that about right?

Reply:
Yes, this is only to suppress warnings for unused anonymous (compiler-generated) identifiers.